src/scenarios/crossgen/test.py

```diff
@@ -8,7 +8,6 @@
 if __name__ == "__main__":
     traits = TestTraits(exename=EXENAME,
                         startupmetric='ProcessTime',
-                        crossgen=True,
```

Comment:
Why are we removing all of these test traits from the default? The ones that we are calling for SoD tests make sense, but why the crossgen test as well?

Reply:
We're removing the concept. The idea was this would guard against trying to do "the wrong thing" with test assets that didn't make sense for a given type of measurement, but it turns out that's not really a thing we need. The crossgen test is maybe a specific counter example, but I think it is very unlikely that anyone is ever really going to try and run things randomly (which was the chief user experience I was trying to protect.)